tests/test_kernel_language.py

```diff
@@ -265,11 +264,10 @@ def kernel(particle, fieldset, time):
 
 
 def random_series(npart, rngfunc, rngargs, mode):
-    random = ParcelsRandom if mode == 'jit' else py_random
-    random.seed(1234)
-    func = getattr(random, rngfunc)
+    ParcelsRandom.seed(1234)
+    func = getattr(ParcelsRandom, rngfunc)
     series = [func(*rngargs) for _ in range(npart)]
-    random.seed(1234)  # Reset the RNG seed
+    ParcelsRandom.seed(1234)  # Reset the RNG seed
```

Comment:
why has this now changed ? This would kind-of defy the purpose of making sure one can - if wanted - use the normal Python RNG, but then just in SciPy mode.

Reply:
Well, I could leave it the old way, but in fact I'd support also using the ParcelsRandom in scipy mode. The old version of this unit test also still worked with the bugfix of course, but I think this new one is tidier